### PR TITLE
2330: SVG colors

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

This PR allows us to set colors on our svgs. Instead of having to use multiple SVGs for different colors, we can now just use styling via styled-components to change colors.

### Proposed changes

- Replace path colors of svgs (mostly black) with `currentColor` -> Displaying only works with setting colors, should be done using our new custom Icon components
- Add react-inlinesvg library to render svgs on web as svg styling is not possible with rendering  components
- All icons now have a default size of 24px but this can be changed using styling with styled-components

### Side effects

- Icons cannot be used without setting color anymore!
- A few icons might look a little different (size and icon itself)

### Resolved issues

Fixes: #2330 

---

### CODE REVIEW

File: assets/icons/website.svg [hunk 1]
```diff
@@ -1 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM4 12c0-.61.08-1.21.21-1.78L8.99 15v1c0 1.1.9 2 2 2v1.93C7.06 19.43 4 16.07 4 12zm13.89 5.4c-.26-.81-1-1.4-1.9-1.4h-1v-3c0-.55-.45-1-1-1h-6v-2h2c.55 0 1-.45 1-1V7h2c1.1 0 2-.9 2-2v-.41C17.92 5.77 20 8.65 20 12c0 2.08-.81 3.98-2.11 5.4z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000">
```
Comment (LeandraH) [inline assets/icons/website.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```
Just to keep the format the same :)

File: native/src/components/NewsHeader.tsx [hunk 2]
```diff
@@ -36,6 +36,11 @@ const HeaderContainer = styled.View`
   margin-bottom: 12px;
 `
 
+const StyledIcon = styled(Icon)`
+  width: 110px;
+  height: 34px;
+`
```
Comment (LeandraH) [inline native/src/components/NewsHeader.tsx:42]: Maybe a comment to explain the magic numbers?

File: native/src/routes/TuNewsDetail.tsx [hunk 3]
```diff
@@ -13,16 +14,13 @@ import urlFromRouteInformation from '../navigation/url'
...
+  margin: 20px 16px 0;
+  border-radius: 8px;
   background-color: rgba(2, 121, 166, 0.4);
 `
-const HeaderImage = styled(TuNewsActiveIcon)`
-  border-top-left-radius: 5px;
-  border-bottom-left-radius: 5px;
+const StyledIcon = styled(Icon)`
+  height: 34px;
+  width: 100px;
```
Comment (LeandraH) [inline native/src/routes/TuNewsDetail.tsx:23]: Same here

**Review by LeandraH — APPROVED** [review]
Very nice! Tested in emulated iOS and Firefox

**Comment by LeandraH** [comment]
Actually, I started getting some errors in the browser console. 

image

image

**Review by f1sh1918 — APPROVED** [review]
Really nice. i like it :)

File: native/src/components/NewsHeader.tsx (hunk 2 above)
Comment (steffenkleinle) [reply native/src/components/NewsHeader.tsx:42]: I don't like the increase of the sizes as I think the tabs are then in relation to the caption too big. Feel free to open a new Issue though. Also, I just took the sizes from before, nothing new here, so I also don't know where the magic numbers are coming from.

File: web/src/components/HeaderNavigationItem.tsx [hunk 4]
```diff
@@ -109,6 +109,11 @@ const Circle = styled.div`
   }
 `
 
+const StyledIcon = styled(Icon)`
```
[the remark this answers is not in the document]
Comment (steffenkleinle) [reply web/src/components/HeaderNavigationItem.tsx:93]: Cood catch!

**Comment by steffenkleinle** [comment]
> Actually, I started getting some errors in the browser console.

Should be fixed now, good catch.

**Pull request merged**